c/scheduling.c

```diff
@@ -35,6 +35,8 @@
 #include "recovery.h"
 #include "scheduling.h"
 #include "openprims.h"
+#include "logging.h"
```

Comment:
Yes, we should avoid doing that. We don't need to change anything since the messages are under the compile time trace flag.